src/setuptools_pyproject_migration/__init__.py

```diff
@@ -239,6 +239,11 @@ def _generate(self) -> Pyproject:
             "version": dist.get_version(),  # TODO try to reverse-engineer dynamic version
         }
 
+        # In older setuptools releases, unspecified license text is replaced with "UNKNOWN"
+        license_text = dist.get_license()
+        if license_text and (license_text != "UNKNOWN"):
+            pyproject["project"]["license"] = dict(text=license_text)
```

Comment:
Almost missed this one, but the same deal about dict literal syntax here. Like the other case, this is a very optional change.

```suggestion
            pyproject["project"]["license"] = {"text": license_text}
```

Reply:
Yeah, I was being lazy I guess. :-)